syntaxes/v.tmLanguage.json

```diff
@@ -1024,186 +860,167 @@
 				}
 			]
 		},
-		"std-types": {
+		"punctuation": {
 			"patterns": [
 				{
-					"name": "storage.type.boolean.v",
-					"match": "\\bbool\\b"
+					"match": "\\.",
+					"name": "punctuation.definition.other.dot.v"
```

Comment:
This should be scoped as `punctuation.delimiter.period.dot.v`; some themes will style `punctuation.definition.` differently on the assumption that the symbol is being used to define a variable (e.g., as in PHP's `$var`).